_security/configuration/security-installation.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: Security installation
+parent: Configuration
+nav_order: 60
+---
+
+# Security installation
+
+Default version of OpenSearch comes with Security features pre-installed, however if this was [disabled]({{site.url}}{{site.baseurl}}/security/configuration/disable/) or OpenSearch Minimum was installed, this page goes through the steps of installing Security features.
```

Comment:
We don't ever refer to the minimal distribution version of OpenSearch "OpenSearch Minimum" in the documentation. Perhaps we can say, ....if the Security plugin was disabled (link) or OpenSearch was installed without security using the minimal distribution method..."

Reply:
Updated as per suggestion